incubator/ibm-blockchain-network/templates/blockchain-services.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "ibm-blockchain-network.name" . }}-ca
+  labels:
+    app: {{ template "ibm-blockchain-network.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    run: {{ template "ibm-blockchain-network.name" . }}-ca
+spec:
+  type: NodePort
+  selector:
+    name: {{ template "ibm-blockchain-network.name" . }}-ca
+    app: {{ template "ibm-blockchain-network.name" . }}
+    release: {{ .Release.Name }}
+  ports:
+  - protocol: TCP
+    port: 7054
+    nodePort: 30000
+    name: grpc
+---
```

Comment:
One resource per file, please. You may use subdirectories to group them.

Reply:
This is fixed.